README.md

```diff
@@ -1,53 +1,69 @@
 
 ## Contributing
 
-Stack-Rook is a community driven project and we welcome contributions.
-See the Crossplane [Contributing](https://github.com/crossplane/crossplane/blob/master/CONTRIBUTING.md) guidelines to get started.
+provider-rook is a community driven project and we welcome contributions. See
+the Crossplane
+[Contributing](https://github.com/crossplane/crossplane/blob/master/CONTRIBUTING.md)
+guidelines to get started.
 
 ## Report a Bug
 
-For filing bugs, suggesting improvements, or requesting new features, please open an [issue](https://github.com/crossplane/stack-rook/issues).
+For filing bugs, suggesting improvements, or requesting new features, please
+open an [issue](https://github.com/crossplane/provider-rook/issues).
 
 ## Contact
 
 Please use the following to reach members of the community:
 
 - Slack: Join our [slack channel](https://slack.crossplane.io)
-- Forums: [crossplane-dev](https://groups.google.com/forum/#!forum/crossplane-dev)
+- Forums:
+  [crossplane-dev](https://groups.google.com/forum/#!forum/crossplane-dev)
 - Twitter: [@crossplane_io](https://twitter.com/crossplane_io)
 - Email: [info@crossplane.io](mailto:info@crossplane.io)
 
 ## Roadmap
 
-Stack-Rook goals and milestones are currently tracked in the Crossplane repository.
-More information can be found in [ROADMAP.md](https://github.com/crossplane/crossplane/blob/master/ROADMAP.md).
+provider-rook goals and milestones are currently tracked in the Crossplane
+repository. More information can be found in
+[ROADMAP.md](https://github.com/crossplane/crossplane/blob/master/ROADMAP.md).
 
 ## Governance and Owners
 
-Stack-Rook is run according to the same [Governance](https://github.com/crossplane/crossplane/blob/master/GOVERNANCE.md) and [Ownership](https://github.com/crossplane/crossplane/blob/master/OWNERS.md) structure as the core Crossplane project.
+provider-rook is run according to the same
+[Governance](https://github.com/crossplane/crossplane/blob/master/GOVERNANCE.md)
+and [Ownership](https://github.com/crossplane/crossplane/blob/master/OWNERS.md)
+structure as the core Crossplane project.
 
 ## Code of Conduct
 
-Stack-Rook adheres to the same [Code of Conduct](https://github.com/crossplane/crossplane/blob/master/CODE_OF_CONDUCT.md) as the core Crossplane project.
+provider-rook adheres to the same [Code of
+Conduct](https://github.com/crossplane/crossplane/blob/master/CODE_OF_CONDUCT.md)
+as the core Crossplane project.
 
 ## Licensing
 
-Stack-Rook is under the Apache 2.0 license.
+provider-rook is under the Apache 2.0 license.
 
-[![FOSSA Status](https://app.fossa.io/api/projects/git%2Bgithub.com%2Fcrossplane%2Fstack-rook.svg?type=large)](https://app.fossa.io/projects/git%2Bgithub.com%2Fcrossplane%2Fstack-rook?ref=badge_large)
+[![FOSSA Status](https://app.fossa.io/api/projects/git%2Bgithub.com%2Fcrossplane%2Fprovider-rook.svg?type=large)](https://app.fossa.io/projects/git%2Bgithub.com%2Fcrossplane%2Fprovider-rook?ref=badge_large)
```

Comment:
will we need to make updates on the FOSSA side or will that be done automatically?

Reply:
I believe this will be done automatically :)